src/LinuxTracing/PerfEventOpen.cpp

```diff
@@ -65,6 +65,7 @@ int mmap_task_event_open(pid_t pid, int32_t cpu) {
   pe.type = PERF_TYPE_SOFTWARE;
   pe.config = PERF_COUNT_SW_DUMMY;
   pe.mmap = 1;
+  pe.mmap_data = 1;
```

Comment:
I am wondering if quick comments here should be added. Like "Enables generation of samples for mmap calls that do not have PROT_EXEC set" (similar for the mmap above). WDYT?

Reply:
Good idea, done.